### ISSUE TITLE
Finger pulse oximetry oxygen saturation lower than blood gas analysis (and a second term for higher)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10671]
**Preferred term label:**
Finger pulse oximetry oxygen saturation lower than blood gas analysis (and a second term for higher)

**Synonyms**

**Definition (free text, please give PubMed ID)**
Inaccurately reduced oxygen saturation as measured by finger or toe pulse oximetry compared to oxygen saturation as measured by venous or arterial blood gas analysis

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Raynaud phenomenon (lower)
Scleroderma (lower)
Carbon monoxide poisoning (higher)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001211
- label: Finger pulse oximetry lower than arterial blood gas analysis measurement of oxygen saturation
- parents: [Abnormal arterial physiology (HP:0025323)](https://hpo.jax.org/app/browse/term/HP:0025323)
- pmid: PMID:38340159
- def: Inaccurately reduced oxygen saturation as measured by finger or toe pulse oximetry compared to oxygen saturation as measured by venous or arterial blood gas analysis.
- comment: This feature may be observed with conditions such as Raynaud phenomenon and scleroderma, in which peripheral microvasculopathy leads to poor perfusion of the fingers.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10671
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001212
- label: Finger pulse oximetry higher than arterial blood gas analysis measurement of oxygen saturation
- parents: [Abnormal arterial physiology (HP:0025323)](https://hpo.jax.org/app/browse/term/HP:0025323)
- pmid: PMID:9792574
- def: Inaccurately elevated oxygen saturation as measured by finger or toe pulse oximetry compared to oxygen saturation as measured by venous or arterial blood gas analysis.
- comment: This feature may be observed with Carbon monoxide poisoning. Presently available pulse oximeters overestimate arterial oxygenation in patients with severe CO poisoning. An elevated COHb level falsely elevates the SaO2 measurements from pulse oximetry.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10671
- synonyms:
